src/test/resources/mappings/instance_collection.json

```diff
@@ -6,7 +6,7 @@
         "url": "/inventory/instances?query=id%3D%3Da89eccf0-57a6-495e-898d-32b9b2210f2f",
         "headers" : {
           "x-okapi-token": {
-            "equalTo": "AAA-BBB-CCC"
+            "equalTo": "eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJtb2QtaW5ucmVhY2giLCJ1c2VyX2lkIjoiMDFjYjFjYjgtZTFiNi00NjNkLWE4OTEtZGU0YzNmMjg1MTkyIiwidHlwZSI6ImFjY2VzcyIsImV4cCI6MTY5MzU3MzQ3NSwiaWF0IjoxNjkzNTcyODc1LCJ0ZW5hbnQiOiJkaWt1In0.IUAZngqq-jEkXtxZlmjDjpiKI-cgSG7YbcfisXlgUVE"
```

Comment:
Can this not be an actual token but rather just a string dummy like "test_token" or similar? I think it can. This looks like a secret is being included in source code, which of course is a big no no. Even if it is not, since presumably this token won't provide access to any existing system, you can't be too careful about this.

Reply:
Agreed. Now I reverted the code with dummy token value